UPGRADE-1.0.md

```diff
@@ -0,0 +1,693 @@
+## 1.0.0
```

Comment:
I would create a `UPGRADE-1.md` including all from 0 -> 1.6.24

I would create a `UPGRADE-2.md` including all from 2.0.0-alpha1 -> 2.x

`UPGRADE.md` start with `3.0.0`

Reply:
Yup that makes sense. Did that.